Source/ThirdParty/libwebrtc/Source/webrtc/webkit_sdk/objc/components/video_codec/RTCVideoDecoderH265.mm

```diff
@@ -329,6 +347,14 @@ - (NSInteger)decodeData:(const uint8_t *)data size:(size_t)size timeStamp:(int64
 }
 
 - (NSInteger)setHVCCFormat:(const uint8_t *)data size:(size_t)size width:(uint16_t)width height:(uint16_t)height {
+  if (auto hvccInformation = ComputeH265InfoFromHVCC(data, size)) {
+    if (hvccInformation->width)
```

Comment:
is this ever going to be non-zero ?

Reply:
This can happen if the HVCC does not have a SPS, which would be strange but possible.